vocab_files/observable_property_concepts/redeployment-observations.ttl

```diff
@@ -2,16 +2,14 @@ PREFIX dcterms: <http://purl.org/dc/terms/>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX schema: <https://schema.org/>
 PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
-PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/nrm/0adc5740-7e20-4509-949c-12ae72500951>
     a skos:Concept ;
     dcterms:source "Laws M, McCallum K, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Camera Traps Module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "A brief summary of the observations made while redployment of the camera traps in the study site. " ;
+    skos:definition "A brief summary of the observations made while redeployment of the camera traps in the study site. " ;
```

Comment:
The definition should be "A brief summary of observations made during the redeployment of camera traps in the study site. "